Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -56,6 +56,13 @@ $appInsightsKey = Get-AppInsightsKey -appInsightsInstrumentationKey $installSett
 $sqlServerAddress = Get-SqlServerAddress -sqlServerAddress $installSettings.sqlServerAddress -installConfigPath $installConfigPath -quiet $quiet
 $identityDatabase = Get-IdentityDatabaseConnectionString -identityDbName $installSettings.identityDbName -sqlServerAddress $sqlServerAddress -installConfigPath $installConfigPath -quiet $quiet
 $metadataDatabase = Get-MetadataDatabaseConnectionString -metadataDbName $installSettings.metadataDbName -sqlServerAddress $sqlServerAddress -installConfigPath $installConfigPath -quiet $quiet
+
+# using methods in DosInstallUtilites to install idpss, which will make it easier to migrate the identity code later 
+$configStore = @{Type = "File"; Format = "XML"; Path = $installConfigPath}
+$idpssConfigStore = Get-DosConfigValues -ConfigStore $configStore -Scope "idpss"
+$commonConfigStore = Get-DosConfigValues -ConfigStore $configStore -Scope "common"
```

Comment:
Should we move the above block closer to the idpss install section?

Reply:
yes